pkgs/os-specific/linux/mdadm/default.nix

```diff
@@ -9,7 +9,10 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-QWcnrh8QgOpuMJDOo23QdoJvw2kVHjarc2VXupIZb58=";
   };
 
-  patches = [ ./no-self-references.patch ];
+  patches = [
+    ./no-self-references.patch
```

Comment:
How about adding comments?
So we know if the patch needs to be removed at the next point release. A few words of explanation would be nice.

Reply:
We can't know _yet_ whether upstream will accept this change or a similar one.

Reply:
Fine, but still a comment should be explaining the reason of the patch, don't you think ?